app/javascript/components/LinkButton.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <button
+    @click="moveMe()"
+    :class="[classmod,'text-white font-bold py-2 px-4 rounded focus:outline-none focus:shadow-outline']"
+    :disabled="fieldDisabled"
+  >{{ fieldPlaceholder }}</button>
```

Comment:
También debiesemos dejar de ponerle a todo "fieldAlgo", creo que usando el mismo nombre es mas intuitivo:
```suggestion
  <button
    @click="moveMe()"
    :class="[classmod,'text-white font-bold py-2 px-4 rounded focus:outline-none focus:shadow-outline']"
    :disabled="disabled"
  >{{ fieldPlaceholder }}</button>
```